students/ScottL/lesson07/RDBMS-master/src/v00_personjob_model.py

```diff
@@ -0,0 +1,107 @@
+"""
+    Simple database example with Peewee ORM, sqlite and Python
+    Here we define the schema
+    Use logging for messages so they can be turned off
+
+    Changelog:
+    2018-05-19: added department for Class Activity07
+"""
+
+import logging
+from peewee import *
```

Comment:
Best not to use `*` imports as other devs will be seeing function names and not know if its part of this lib or something you created in another class somewhere or something and they'll be hunting for the origin of it.